src/libusermode/printers/printers.cpp

```diff
@@ -143,8 +143,8 @@ std::string escape_str(const std::string& s)
     return os.str();
 }
 
-ArgumentPrinter::ArgumentPrinter(std::string arg_name, bool print_no_addr, numeric_format_t base) :
-    name(arg_name), print_no_addr(print_no_addr), numeric_format(base)
+ArgumentPrinter::ArgumentPrinter(const PrinterConfig& config, std::string arg_name) :
+    config(config), name(arg_name)
```

Comment:
```suggestion
ArgumentPrinter::ArgumentPrinter(std::string arg_name, PrinterConfig config = PrinterConfig{}) :
    config(config), name(arg_name)
```
By reversing those, we can call `ArgumentPrinter{"ApiName"}` without having to specify `PrinterConfig` if we want to use default one :) In theory we could `std::move` it, but I believe it fits into 1 register.